packages/web/src/components/incentivize/incentivize-block-tooltip-content/IncentivizeBlockTooptipContent.tsx

```diff
@@ -0,0 +1,55 @@
+import { useMemo } from "react";
+import * as S from "./IncentivizeBlockTooltipContent.styles";
+
+type Props = {
+  latestBlockHeight?: string;
+  period: number;
+};
+
+function IncentivizeBlockTooltipContent({ latestBlockHeight, period }: Props) {
+  const tier = useMemo(() => {
+    switch (period) {
+      case 180:
+        return 15552000;
+      case 365:
+        return 31536000;
+      case 90:
+      default:
+        return 7776000;
+    }
+  }, [period]);
```

Comment:
Is it the number of blocks created by date?
I think you calculated the block generation time as 1 second, is that correct ?

Also, unless you're looking for the average block creation time, it seems like a good idea to keep the block average production time static.

Reply:
![image](https://github.com/user-attachments/assets/92035cb4-a5b0-4a6b-acac-1ca57bf1568e)

Reply:
Commented the code out for now